src/constants.js

```diff
@@ -52,3 +52,7 @@ export const CUSTOM_FILTERS = {
         }
     }
 };
+
+// Query params for state filters
+export const WAITING_SYSTEMS_VALUE = 'waiting';
+export const SUGGESTIONS_SYSTEMS_VALUE = 'suggestions';
```

Comment:
Just suggestion

```suggestion
export const STATE_LIST = ['Undersized', 'Oversized', 'Under pressure', 'Idling', 'Optimized', 'Waiting for data']
```
Example - 
In case of `with_suggestions=true` you can use it like -  `STATE_LIST.filter(state => !['Optimized', 'Waiting for data'].includes(state))`

Reply:
Hello @kgaikwad thank you for the suggestion, but I would prefer directly passing the array with state values (current implementation) to set the state filter values as I feel it makes the code more readable. Let me know what you think.

Reply:
Sure @PreetiW! My intention behind this suggestion was to have a constant in file so that one can easily add/remove  state value from list when `with_suggestions=true` without looking at implementation.

Reply:
hmm I see what you saying here @kgaikwad. So for `with_suggestions=true` we use the list as you suggested above and for for `with_waiting_for_data=true` we assign the filter array value directly since it is only one value ['Waiting for data'] did I get it right?

Reply:
My suggestion was to have one single constant which includes all states.
- when `with_suggestions=true`,  assign `STATE_LIST.filter(state => !['Optimized', 'Waiting for data'].includes(state))` 
- when `with_waiting_for_data=true`, assign ['Waiting for data'] directly as it will always be a single state value.

Depending upon code readability, please feel free to ignore as it was just a suggestion.